spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -602,6 +590,20 @@ def attempt_to_bypass_2fa
           expect(page).to have_current_path(account_path)
         end
       end
+
+      def simulate_platform_authenticator_available
+        # Since the element will have already initialized by this point and it can't be guaranteed
+        # that isUserVerifyingPlatformAuthenticatorAvailable would yield the expected value, stub
+        # and reconnect the elements to simulate as if it were the expected value on page load.
+        stub_platform_auth_available_js = <<~JS
+          window.PublicKeyCredential.isUserVerifyingPlatformAuthenticatorAvailable = () => Promise.resolve(true);
+        JS
+        page.evaluate_script(stub_platform_auth_available_js)
+        reconnect_webauthn_input_js = <<~JS
+          document.querySelectorAll('lg-webauthn-input').forEach((input) => input.connectedCallback());
+        JS
+        page.evaluate_script(reconnect_webauthn_input_js)
```

Comment:
personally I wouldn't bother with naming these strings, I'd just inline them? maybe even combine into one step?

```suggestion
        page.evaluate_script(<<~JS)
          window.PublicKeyCredential.isUserVerifyingPlatformAuthenticatorAvailable = () => Promise.resolve(true);
          document.querySelectorAll('lg-webauthn-input').forEach((input) => input.connectedCallback());
        JS
```

Reply:
But I'll inline the strings at least, good call.